### PR TITLE
Adds ability to set last reconciliation date to MockPumpManager

### PR DESCRIPTION
**Pull request by rickpasetto**
This allows for being able to more easily test stale pump data.  (You'll need to delete any pump data first)

### CODE REVIEW

File: MockKit/MockPumpManager.swift [hunk 1]
```diff
@@ -53,7 +53,6 @@ public final class MockPumpManager: TestingPumpManager {
     )
 
     private static let deliveryUnitsPerMinute = 1.5
-    private static let pulsesPerUnit: Double = 20
```
Comment (rickpasetto) [inline MockKit/MockPumpManager.swift:56]: This was an unused variable so I deleted it...

**Review by nhamming — APPROVED** [review]
LGTM

**Review by ps2 — APPROVED** [review]
LGTM.  It looks like once set, it can't be unset? I guess this is fine; if you need to test out-of-date pump data and then want to revert to normal behavior you can just restart the app.

**Comment by rickpasetto** [comment]
>  It looks like once set, it can't be unset?

@ps2 yeah, good catch.  I'll add something to make it possible.

**Pull request merged**